<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QMainWindow" name="login">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>login</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>671</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Receptionist</string>
    </property>
    <widget class="QPushButton" name="pushButton_loginR">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>LogIn</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_loginAsAdminR">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>LogIn as Admin</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_createNewR">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>100</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Create new account</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_usernameR">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>58</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600; color:#000000;&quot;&gt;Username : &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_usernameR">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>361</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_passwordR">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>361</width>
       <height>20</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_passwordR">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600; color:#000000;&quot;&gt;Password : &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_imgR">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>20</y>
       <width>181</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string>          image of recpetionistl</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>671</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Pharmacist</string>
    </property>
    <widget class="QPushButton" name="pushButton_loginP">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>LogIn</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_loginAsAdminP">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>LogIn as Admin</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_createNewP">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>100</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Create new account</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>58</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600; color:#000000;&quot;&gt;Username : &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600; color:#000000;&quot;&gt;Password : &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_passwordP">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>361</width>
       <height>20</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_usernameP">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>361</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_imgP">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>10</y>
       <width>201</width>
       <height>151</height>
      </rect>
     </property>
     <property name="text">
      <string>                image of pharmecy </string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
